de.fabrik19.mos.Launcher.yml

```diff
@@ -0,0 +1,38 @@
+app-id: de.fabrik19.mos.Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: mos-launcher
+modules:
+  - name: mos-launcher
+    buildsystem: simple
+    build-options:
+      strip: false
+      no-debuginfo: true
+    build-commands:
+      - mkdir -p /app/bin
+      - cp libHarfBuzzSharp.so /app/bin/libHarfBuzzSharp.so
+      - cp libSkiaSharp.so /app/bin/libSkiaSharp.so
+      - cp mos-launcher /app/bin/mos-launcher
+      - cp mos-launcher.pdb /app/bin/mos-launcher.pdb
+      - cp mos-launcher.xml /app/bin/mos-launcher.xml
+      - mkdir -p /app/share/icons/hicolor/scalable/apps/
+      - cp icon.svg /app/share/icons/hicolor/scalable/apps/de.fabrik19.mos.Launcher.svg
+      - mkdir -p /app/share/applications/
+      - cp de.fabrik19.mos.Launcher.desktop /app/share/applications/de.fabrik19.mos.Launcher.desktop
+      - mkdir -p /app/share/metainfo/
+      - cp de.fabrik19.mos.Launcher.metainfo.xml /app/share/metainfo/de.fabrik19.mos.Launcher.metainfo.xml
+    sources:
+      - type: archive
+        url: https://download.mobilitysuite.de/launcher/linux-x64-2.0.7.tar.gz
```

Comment:
how is this gonna work on aarch64 that you still build?

Reply:
also the download URL seems to be unrelated with the app-id. Can you explain?

Reply:
Added an `arm64` version.
The download URL is different, because it is a product of Fabrik19. If it would be better to move this URL to the fabrik19 domain, we could just move it.